extensions/amp-story/1.0/amp-story.js

```diff
@@ -1623,6 +1623,9 @@ export class AmpStory extends AMP.BaseElement {
               mutation => mutation.attributeName === 'open')) {
             this.storeService_.dispatch(Action.TOGGLE_SIDEBAR,
                 this.sidebar_.hasAttribute('open'));
+          } if (mutationsList.some(
```

Comment:
Can we refactor this so we only loop once through the list of mutations?

Reply:
Done